jerry-debugger/jerry-client-ws.py

```diff
@@ -560,6 +563,31 @@ def send_message(self, message):
                 message = message[bytes_send:]
             size -= bytes_send
 
+    def delete_active(self):
+        for i in self.active_breakpoint_list.values():
+            breakpoint = self.active_breakpoint_list[i.active_index]
+            del self.active_breakpoint_list[i.active_index]
+            breakpoint.active_index = -1
+            self.send_breakpoint(breakpoint)
+
+    def delete_pending(self):
+        for i in self.pending_breakpoint_list.values():
+            del self.pending_breakpoint_list[i.index]
+
+    def breakpoint_exists(self, breakpoint):
+        for i in self.pending_breakpoint_list:
+            if getattr(breakpoint, "function", None) is None:
+                if not getattr(self.pending_breakpoint_list[i], "line", None) is None:
+                    if self.pending_breakpoint_list[i].line == breakpoint.line:
```

Comment:
No, since you should always give the arg in source:line format, so if the line is given, there should be a source.